package.json

```diff
@@ -595,6 +595,11 @@
                     "default": false,
                     "description": "Enables remote debugging of a Web App. This experimental preview only supports node on Linux."
                 },
+                "appService.deploymentConfigurations": {
+                    "scope": "resource",
+                    "type": "object",
```

Comment:
You should make the type more specific than 'object'. Users should be able to manually set their deployment configuration just by reading the setting descriptions/types